script/Arcueid/Day 13/13_00_ARC13_1B.txt

```diff
@@ -1636,6 +1642,7 @@ No, of course not.
 [sha:db47cc9d6e32e9e452ce99a8a05907fb771c16fd]{
 -- Page 208, Offset 32233.
 -- 「知るか。言う事をきかないなら、ここでメチャクチャに暴れてやる」
+// CRINGE MOMENT
```

Comment:
this sounds god awful in english. Maybe the equally cringe but less awkward "I'll fuck you up."